osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -151,7 +151,7 @@ protected PlaceholderState PlaceholderState
                         break;
 
                     case PlaceholderState.NotLoggedIn:
-                        replacePlaceholder(new LoginPlaceholder());
+                        replacePlaceholder(new LoginPlaceholder(@"Please sign in to view online leaderboards!"));
```

Comment:
seems unrelated to this implementation.

Reply:
it is actually, the `LoginPlaceholder` constructor now has a message parameter.